src/PortlandLabs/ConcreteCmsTheme/Navigation/HeaderNavigationFactory.php

```diff
@@ -24,21 +24,12 @@ public function createNavigation(): NavigationInterface
     {
         $activeSection = $this->activeSection;
         // Determine if this navigation menu is for the concretecms.org site or not
-        $siteTarget = $_ENV['URL_SITE_MARKETING_OPENSOURCE'];
-        $siteDetected = \Core::make('site')->getSite()->getSiteCanonicalURL();
-        // remove trailing slash if exists
-        if (strcmp('/', substr($siteDetected, -1)) === 0) {
-            $siteDetected = substr($siteDetected, 0, -1);
-        }
-        $siteIsMarketingOpensource = false;
-        if (strcmp($siteTarget, $siteDetected) === 0) {
-            $siteIsMarketingOpensource = true;
-        }
+        $enableDarkMode = \Core::make('site')->getSite()->getConfigRepository()->get("concrete_cms_theme.enable_dark_mode");
```

Comment:
This class uses application aware trait/interface so you can use `$this->app` instead of `\Core`
```suggestion
        $enableDarkMode = $this->app->make('site')->getSite()->getConfigRepository()->get("concrete_cms_theme.enable_dark_mode");
```